engine-nextgen/src/main/kotlin/graphql/nadel/enginekt/blueprint/NadelFieldInstruction.kt

```diff
@@ -76,6 +77,12 @@ interface NadelGenericHydrationInstruction {
      * or [NadelBatchHydrationMatchStrategy.MatchObjectIdentifier.sourceId].
      */
     val sourceFields: List<NadelQueryPath>
+
+    /**
+     * The field definition in the overall schema referenced by [queryPathToActorField].
+     * should be non-null in the future
+     */
+    val overallActorFieldDef: GraphQLFieldDefinition?
```

Comment:
previously I added this to the batch instruction only, now adding it to `NadelGenericHydrationInstruction`